metered/src/metric.rs

```diff
@@ -61,6 +61,9 @@ impl<'a, R, M: Metric<R>> Drop for ExitGuard<'a, R, M> {
 pub trait Counter: Default + Clear + Serialize {
     /// Increment the counter
     fn incr(&self);
+
+    /// Returns true if the counter is set to 0.
+    fn is_cleared(&self) -> bool;
```

Comment:
Should this be defined in the `Clear` trait, or even a separate trait?  I see the problem with implementing this for Histograms: 0 can be a measured value on some histograms.

Reply:
That makes sense, I've taken this out into a separate `Clearable` trait.